libdebug/architectures/aarch64/aarch64_breakpoint_validator.py

```diff
@@ -0,0 +1,24 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2024 Roberto Alessandro Bertolini. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from libdebug.data.breakpoint import Breakpoint
+
+
+def validate_breakpoint_aarch64(bp: Breakpoint) -> None:
+    """Validate a hardware breakpoint for the AARCH64 architecture."""
+    if bp.condition.lower() not in ["r", "w", "rw", "x"]:
```

Comment:
why don't directly save the condition in lower case?

Reply:
The condition is already enforced to be lowercase at this point of the code, I was just making sure...